src/loaders/compute_tools/tool_common.py

```diff
@@ -362,7 +364,9 @@ def _create_metadata_file(
         for genome_id, genome_meta_info in meta.items():
```

Comment:
we should probably consider adding a header row as more columns are added. But that can be future PRs when you refactor the parser script since adding the header will also require some changes for the parser script.

Reply:
Good advice!! Header concern will be addressed in PR3 as described in [RE2022-157](https://kbase-jira.atlassian.net/jira/software/projects/RE2022/boards/78?selectedIssue=RE2022-157)